### ISSUE TITLE
idea run configurations should include assertions

### ISSUE THREAD

**Issue by carterkozak** [palantir/gradle-baseline#624]
## What happened?

Assertions are never triggered running from source, only by the gradle `test` task.

## What did you want to happen?

Running from source in the IDE should include the `-ea` flag. At least for tests, but preferably everywhere.

**Comment by uschi2000** [comment]
We haven’t historically encouraged the use of assertions, why would you like to change this, Carter?

**Comment by carterkozak** [comment]
The problem is that we run tests with assertions enabled, causing functional differences between CI nodes and running locally in the IDE.

**Comment by markelliot** [comment]
seems like the fix would be ensuring _tests_ in idea run with -ea by default (this is true for Eclipse IIRC)

**Comment by carterkozak** [comment]
Yes, I don't have much preference elsewhere. For what it's worth, this is the default for intellij, but I think our gradle-baseline idea configuration overrides it in some way which removes the `-ea` flag.

**Comment by uschi2000** [comment]
Alternatively, should be ban assertions in all code bases?
